x-pack/plugins/index_management/__jest__/client_integration/home/data_streams_tab.test.ts

```diff
@@ -449,4 +460,78 @@ describe('Data Streams tab', () => {
       expect(tableCellsValues).toEqual([['', 'non-managed-data-stream', 'green', '1', 'Delete']]);
     });
   });
+
+  describe('delete data stream privileges', () => {
```

Comment:
The title is a bit misleading as it seems that we will delete the privileges :) What about simply: "Privileges: 'delete data stream'"